src/content/docs/release-notes/docs-release-notes/docs-6-28-2024.mdx

```diff
@@ -0,0 +1,74 @@
+---
+subject: "Docs"
+releaseDate: '2024-06-28'
+version: 'version: June 21-27, 2024'
+---
+
+### New docs
+
+* Mobile monitoring's [application not responding](/docs/mobile-monitoring/mobile-monitoring-ui/application-not-responding) helps you track and analyze when and why your Android application's blocked for more than five seconds.
+* New browser monitoring [log](/docs/browser/new-relic-browser/browser-apis/log) and [wrapLogger](/docs/browser/new-relic-browser/browser-apis/wraplogger) API docs define the syntax, requirements, parameters, and examples on how to use these new endpoints.
+
+### Major changes
+
+* We've removed our top-level `Other capabilities` category and migrated everything that was there into more appropriate categories. This should make it somewhat easier for you to find what you're looking for.
```

Comment:
`somewhat easier` feels pretty wishy-washy. :) Let's sell it.

Reply:
Agreed!